config/config.go

```diff
@@ -917,6 +918,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	// for them and specify all the parameters they need for them to work correctly.
 	v.SetDefault("adapters.33across.endpoint", "http://ssc.33across.com/api/v1/hb")
 	v.SetDefault("adapters.33across.partner_id", "")
+	v.SetDefault("adapters.acuityads.endpoint", "http://{{.Host}}.admanmedia.com/bid?token={{.AccountID}}")
```

Comment:
Why does AcuityAds require a per-request sub domain?

This seems to be a growing trend among adapters, which is worrying because it will reduce server performance with connection pool starving. This will impact the amount of traffic AcuityAds will receive from PBS hosts.

Reply:
Hello, @SyntaxNode ! Thanks for your comments! Main reason for this step is that we have some different geographical located datacenters in order to reduce latency for our partners. Currently we work for implement geo-dns for our project. I`ll send pull request as soon as we switch our current logic.